_developers/release_guide.md

```diff
@@ -257,41 +260,45 @@ mvn deploy -DskipTests -Ppassphrase -Prelease
 
-15. Upload the release to [Apache dev released repository](https://dist.apache.org/repos/dist/dev/incubator/servicecomb/incubator-servicecomb-saga/).
+4. Upload the release to  directory created in last step.
 
-16. Download all the releases from SVN and verify the signature and checksum.
+5. Download all the releases from SVN and verify the signature and checksum.
 
 ***PPMC approval***
 
-17. Send the voting mail in dev@servicecomb.apache.org.
+1. Send the voting mail in dev@servicecomb.apache.org.
 
-18. Wait for 72 hours or unless you get 3 +1 binding vote with no -1 vote. If you get even one -1 binding vote then fix the issue and start again from Step 1.
+2. Wait for 72 hours or unless you get 3 +1 binding vote with no -1 vote. If you get even one -1 binding vote then fix the issue and start again from Step 1.
 
-19. Publish the result of the vote in dev@servicecomb.apache.org.
+3. Publish the result of the vote in dev@servicecomb.apache.org.
 
 ***IPMC approval***
 
-20. Send the voting mail in general@incubator.apache.org
+1. Send the voting mail in general@incubator.apache.org
 
-21. Wait for 72 hours or unless you get 3 +1 binding vote with no -1 vote. If you get even one -1 binding vote then fix the issue and start again from Step 1.
+2. Wait for 72 hours or unless you get 3 +1 binding vote with no -1 vote. If you get even one -1 binding vote then fix the issue and start again from Step 1.
 
-22. Publish the result of the vote in general@incubator.apache.org.
+3. Publish the result of the vote in general@incubator.apache.org.
 
 ***Announcements***
 
-23. Upload the releases to [Apache release repository](https://dist.apache.org/repos/dist/release/incubator/servicecomb/incubator-servicecomb-saga/).
+1. Upload the releases to [Apache release repository](https://dist.apache.org/repos/dist/release/incubator/servicecomb/incubator-servicecomb-saga/).
 
 
-24. Wait for 24 hours to replicate the release in all the mirrors.
+2. Wait for 24 hours to replicate the release in all the mirrors.
 
-25. Upload the release page of ServiceComb Website.
+3. Upload the release page of ServiceComb Website.
 
-26. Send the announcement mails to dev@servicecomb.apache.org, general@incubator.apache.org, announce@apache.org
+4. Send the announcement mails to dev@servicecomb.apache.org, general@incubator.apache.org, announce@apache.org
 
 
 **NOTE**
-The whole process generally takes 2 weeks to complete assuming you don't get any -1 from PPMC and IMPC, so please plan the release activity before hand.
+ - The whole process generally takes 2 weeks to complete assuming you don't get any -1 from PPMC and IMPC, so please plan the release activity before hand.
+ - Make sure to clear all the release candidates from the dev-svn after the publishing the releases.
```

Comment:
Not sure if we need to clean up the release candiates if we add the rc version for it. 
Could you double check it?